pkg/inttest/http.go

```diff
@@ -171,6 +171,41 @@ func (hc *HTTPClient) PostJSON(t *testing.T, path string, requestBody io.Reader,
 	require.NoError(t, err, errMsg+": failed to unmarshal response body")
 }
 
+func (hc *HTTPClient) SignIn(t *testing.T, email, password string) (*http.Cookie, *http.Cookie) {
+	t.Helper()
+
+	req := hc.NewRequest(t, http.MethodPost, "/tokens", nil, WithHeader("Content-Type", "application/json"), WithBasicAuth(email, password))
+	res := hc.do(t, req)
+	require.Zero(t, res.ContentLength)
```

Comment:
some message saying what we expected is useful in those requires hidden in the client (same below with finding cookies)